src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -102,14 +102,14 @@
   </xsl:param>
 
   <xsl:param name="title.font.family">
-    <xsl:call-template name="pickfont-serif"/>
+    <xsl:call-template name="pickfont-sans"/>
   </xsl:param>
 
   <!--
     Text properties
   -->
 
-  <xsl:param name="hyphenate">false</xsl:param>
+  <xsl:param name="hyphenate">true</xsl:param>
```

Comment:
Question: Isn't it so, that if you do not install the additional hyphenation-patterns then `<xsl:param name="hyphenate">false</xsl:param>` is silently ignored anyway?
After installing the patterns, it took me some time to find this switch, I would prefer it works out of the box.